main.go

```diff
@@ -158,6 +160,26 @@ func linkJiraIssueToCurrentBranch() {
 	printInfoToConsole(string(output))
 }
 
+func unlinkJiraIssueFromCurrentBranch() {
+	currentBranchName, err := git.GetCurrentBranchName()
+	if err != nil {
+		printErrorToConsole(err)
+		return
+	}
+
+	issueKey := config.GetIssueKey()
+	updatedBranchName := git.RemoveIssueKeysFromBranchName([]string{issueKey}, currentBranchName)
+
+	output, err := git.UpdateCurrentBranchName(updatedBranchName)
+	if err != nil {
```

Comment:
What do you think to move error check to utils or something like that?

Reply:
I though about, but as we do not have any additional functionality like logging errors to external service or similar and in some cases we may return immediately and in other continue - I didn't find any good abstraction that solves current 'code duplication' :)